.github/workflows/cdelivery-ecs-library-caller.yml

```diff
@@ -28,9 +28,8 @@ jobs:
       ecs-cluster: ${{ secrets.ECS_CLUSTER_GRAASP_EXPLORE_STAGE }}
       ecs-service: ${{ secrets.ECS_SERVICE_GRAASP_EXPLORE_STAGE }}
       next-public-api-host: ${{ secrets.NEXT_PUBLIC_API_HOST_STAGE }}
-      next-public-domain: ${{ secrets.STAGE_GRAASP_DOMAIN }}
+      next-public-domain: ${{ secrets.REACT_APP_DOMAIN_STAGE }}
```

Comment:
Is it expected that the variables do not start with the usual `NEXT_` prefix in the secrets?

Reply:
Yes this is because we are using Org-level secrets that refer to `graasp.org` or `dev.graasp.org` etc ... so they were created with React projects in mind, this is why they do not really have the naming convention...

Reply:
And this is only on our CI/CD pipeline side, the local variables are all `NEXT_` prefixed as usual.